bin/docker_push

```diff
@@ -10,6 +10,7 @@ echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdi
 docker tag "${TRAVIS_REPO_SLUG}:latest" "${DOCKER_IMAGE}"
 
 if [[ "${CI}" == "true" ]]; then
+    docker push "${TRAVIS_REPO_SLUG}:latest"
```

Comment:
If you just push the name of the repository, without any tag, it will push all the local tags. That way you don't have to push twice.

Reply:
> If you just push the name of the repository, without any tag, it will push all the local tags. That way you don't have to push twice.

Didn't know that. I'll change the script to do that.